docs/pages/web/button.js

```diff
@@ -523,10 +525,19 @@ function SemiTransparentWhiteButtonExample() {
         <MainSection.Subsection
           title="Icons"
           description={`
-Icon end
-Adds an icon after the Button text. Icons should only be used to visually reinforce a specific function or interaction of the Button. Menus and external links are a common use case. Use \`visit\` when linking to an external URL or \`arrow-down\` when displaying a Popover on click. Note that iconEnd on Button is not accessible to screen readers.
+\`iconEnd\` adds an icon after the Button text. Icons should only be used to visually reinforce a specific function or interaction of the Button. Menus and external links are a common use case. Use \`visit\` when linking to an external URL or \`arrow-down\` when displaying a Popover on click. Note that iconEnd on Button is not accessible to screen readers.
 `}
         >
+          <SlimBanner
+            type="recommendationBare"
+            iconAccessibilityLabel="Recommendation"
+            message="Use Gestalt's Eslint rule to enforce the correct icons usage in Button."
```

Comment:
nit: according to [their docs](https://eslint.org/), it's usually styled as "ESLint"